src/Symfony/Component/Validator/Constraints/YamlValidator.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use Symfony\Component\Yaml\Exception\ParseException;
+use Symfony\Component\Yaml\Parser;
+
+/**
+ * @author symfonyaml
```

Comment:
you should add the whole GitHub profile url here

Reply:
Added `@author Kev <https://github.com/symfonyaml>` in the 4 following files :
 - src/Symfony/Component/Validator/Constraints/Yaml.php
 - src/Symfony/Component/Validator/Constraints/YamlValidator.php
 - src/Symfony/Component/Validator/Tests/Constraints/YamlTest.php
 - src/Symfony/Component/Validator/Tests/Constraints/YamlValidatorTest.php

Thank you